cartographer/cloud/internal/client_server_test.cc

```diff
@@ -178,6 +179,22 @@ class ClientServerTest : public ::testing::Test {
     EXPECT_TRUE(stub_for_uploading_server_ != nullptr);
   }
 
+  void SetOptionsToTSDF2D() {
+    trajectory_builder_options_.mutable_trajectory_builder_2d_options()
+        ->mutable_submaps_options()
+        ->mutable_range_data_inserter_options()
+        ->set_range_data_inserter_type(
+            ::cartographer::mapping::proto::RangeDataInserterOptions::
+                TSDF_INSERTER_2D);
+    trajectory_builder_options_.mutable_trajectory_builder_2d_options()
+        ->mutable_submaps_options()
+        ->mutable_grid_options_2d()
+        ->set_grid_type(::cartographer::mapping::proto::GridOptions2D::TSDF);
+    trajectory_builder_options_.mutable_trajectory_builder_2d_options()
+        ->mutable_ceres_scan_matcher_options()
+        ->set_occupied_space_weight(10.0);
```

Comment:
this differs from map_builder_test, intended?

Reply:
It's unified now.